IPython/html/static/widgets/js/widget_box.js

```diff
@@ -4,10 +4,41 @@
 define([
     "widgets/js/widget",
     "jqueryui",
+    "underscore",
     "base/js/utils",
     "bootstrap",
-], function(widget, $, utils){
+], function(widget, $, _, utils){
     "use strict";
+    var unpack_models = function unpack_models(value, model) {
```

Comment:
You should probably export this function so custom widget authors don't have to rewrite this function.